defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+# NB. Supported RHEL versions: RHEL 8+
+#
+# Enable or disable service after configuration
+# to allow testing and verification before use.
+# NB. See ima_evm_setup if planning to use IMA.
+fapolicyd_setup_enable_service: false
+
+# trust list for fapolicyd configuration file
+fapolicyd_setup_trust: ""
```

Comment:
```yaml
fapolicyd_setup_trust: "{{ null if ansible_facts['distribution_version'] is version('8.4', '<') else 'rpmdb,file' }}"
```
something like that

Reply:
I'd prefer to keep it clear and to use explicit to '""' instead of 'none', and then check for the variable to be 'length == 0'.
And don't forget to check for 'ansible_facts.os_family == RedHat' something like this.